portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACPagePath.macro

```diff
@@ -0,0 +1,54 @@
+definition {
+
+	@summary = "Clears segment search filter"
+	macro clearSegmentSearchFilter() {
+		Type.sendKeys(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "keys=CONTROL,a");
+
+		KeyPress(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "\BACK_SPACE");
+	}
+
+	@summary = "Selects the segment in page filter"
+	macro clickFilterSegment(segmentName = null) {
+		Click(
+			key_segmentName = ${segmentName},
+			locator1 = "ACUtils#PAGE_SEGMENT_FILTER");
+	}
+
+	@summary = "Removes the segment in page filter"
+	macro removeSegmentFilter() {
+		Click(locator1 = "ACSites#REMOVE_SEGMENT_FILTER");
+	}
+
+	@summary = "Asserts the total of page path"
+	macro viewPagePathTotal(pageTotal = null) {
+		var key_pageTotal = ${pageTotal};
+
+		AssertElementPresent(
+			locator1 = "ACSites#PAGE_PATH_TOTAL",
+			value1 = ${pageTotal});
```

Comment:
Hey @joshitagaki, I think the `AssertElementPresent` function does not use `value1` to compare the text found by xpath with the value you are using in the variable. So I think you can delete the `value1` here. What do you think?

```suggestion
		AssertElementPresent(locator1 = "ACSites#PAGE_PATH_TOTAL");
```

Reply:
Thanks I changed it to AssertTextEquals instead